manifests/init.pp

```diff
@@ -26,6 +26,7 @@
   String $config_group = $fluentd::params::config_group,
   Hash $configs = $fluentd::params::configs,
   Hash $plugins = $fluentd::params::plugins,
+  Boolean $purge_config_dir = $fluentd::params::manage_config_dir
```

Comment:
Looks like you need to rename this to `purge_config_dir`

```suggestion
  Boolean $purge_config_dir = $fluentd::params::purge_config_dir
```

Reply:
Sorry for that miss, I should drink more coffee to be awake :laughing: